README.md

```diff
@@ -85,18 +85,68 @@ For example
 npx i18n-translate-generator translate "Hello world" "en" "greetings" -s "example/custom-setting.config.json"
 ```
 
+### Nested Translations
+
+You can nest translations by using the `.` character on the property name parameter, for example:
+
+```sh
+npx i18n-translate-generator translate "Account settings" "en" "accountSettings.title"
```

Comment:
Maybe as an example, use Spanish, as is not noticeable English to English

Reply:
Good point, will take a look